shared/domains/chelonia/chelonia.js

```diff
@@ -1642,50 +1646,110 @@ export default (sbp('sbp/selectors/register', {
   // a general rule, you shouldn't be calling this selector directly unless
   // you're building a utility library or if you have very specific needs. In
   // this case, see if `chelonia/kv/queuedSet` covers your needs.
+  // `data` is allowed to be falsy, in which case a fetch will occur first and
```

Comment:
If `data` is allowed to be falsey, then the Flowtype info should be updated below